content/intro/index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: आरंभ करना
+description: Brigade(ब्रिगेड) के साथ काम कैसे शुरू करें
```

Comment:
Nit, space before `(`

Reply:
When we have `English (Hindi)` translation, we can do only once. From next time, we can stick to `English`.

Reply:
So am not changing this now. Is that fine?

Reply:
Fixed it.